.github/workflows/docker-publish.yml

```diff
@@ -28,6 +28,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
+        with:
+          node-version: 16.16
```

Comment:
How about using `lts/gallium` ? This will make sure you're always running on the latest v16 release with security fixes

Reply:
Thanks for the tip. Done!